services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -187,6 +188,15 @@ export const getActionDetails = (
                     '"',
                 application: action.deleteEnvFromApp.application,
             };
+        case 'releaseTrain':
+            return {
+                type: ActionTypes.ReleaseTrain,
+                name: 'Release Train',
+                dialogTitle: 'Are you sure you want to run a Release Train',
+                tooltip: '',
+                summary: 'Run release train for environment ' + action.releaseTrain.target,
```

Comment:
When talking about release train environments, please use either "from" or "to":
```suggestion
                summary: 'Run release train to environment ' + action.releaseTrain.target,
```